openstack/sharedfilesystems/v2/sharenetworks/urls.go

```diff
@@ -9,3 +9,7 @@ func createURL(c *gophercloud.ServiceClient) string {
 func deleteURL(c *gophercloud.ServiceClient, id string) string {
 	return c.ServiceURL("share-networks", id)
 }
+
+func listURL(c *gophercloud.ServiceClient) string {
+	return c.ServiceURL("share-networks", "detail")
```

Comment:
Not necessarily, but this isn't the `List` URL, right? It's the `List Detail` URL. We should name the operation `ListDetail` (and this function `listDetailURL`) if it's returning the full list response.